src/main/java/appeng/recipes/game/WirelessTerminalRecipe.java

```diff
@@ -0,0 +1,145 @@
+package appeng.recipes.game;
+
+import appeng.api.AEApi;
+import appeng.api.definitions.IComparableDefinition;
+import appeng.api.definitions.IDefinitions;
+import net.minecraft.init.Items;
+import net.minecraft.inventory.IInventory;
+import net.minecraft.inventory.InventoryCrafting;
+import net.minecraft.item.ItemStack;
+import net.minecraft.item.crafting.IRecipe;
+import net.minecraft.nbt.NBTTagCompound;
+import net.minecraft.world.World;
+import net.minecraftforge.oredict.OreDictionary;
+
+import javax.annotation.Nullable;
+import java.util.ArrayList;
+
+/**
+ * Created by Tom on 10/2/2015.
+ */
+public class WirelessTerminalRecipe implements IRecipe {
+
+    private final IComparableDefinition terminal;
+    private final ItemStack waterBucket;
+
+    private static final ArrayList<String> dyes = new ArrayList<String>();
+    static {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this left curly brace to the beginning of next line of code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3ALeftCurlyBraceStartLineCheck)